aiomisc/io.py

```diff
@@ -1,36 +1,74 @@
 import asyncio
-from functools import partial
+from functools import partial, total_ordering
 
 from .thread_pool import threaded
 
 
 opener = threaded(open)
```

Comment:
Maybe, add it as constant to `AsyncFileIOBase`? This will allow overriding it in subclasses.